concordium-consensus/src/Concordium/GlobalState/DummyData.hs

```diff
@@ -345,6 +345,29 @@ dummyRewardParametersV2 =
                 }
         }
 
+dummyRewardParametersV3 :: RewardParameters 'ChainParametersV3
+dummyRewardParametersV3 =
+    RewardParameters
+        { _rpMintDistribution =
+            MintDistribution
+                { _mdMintPerSlot = CFalse,
+                  _mdBakingReward = AmountFraction 60000, -- 60%
+                  _mdFinalizationReward = AmountFraction 30000 -- 30%
+                },
+          _rpTransactionFeeDistribution =
+            TransactionFeeDistribution
+                { _tfdBaker = AmountFraction 45000, -- 45%
+                  _tfdGASAccount = AmountFraction 45000 -- 45%
+                },
+          _rpGASRewards =
+            GASRewards
+                { _gasBaker = AmountFraction 25000, -- 25%
+                  _gasFinalizationProof = CFalse,
+                  _gasAccountCreation = AmountFraction 200, -- 0.2%
+                  _gasChainUpdate = AmountFraction 50 -- 0.05%
+                }
+        }
+
```

Comment:
I think this could benefit from some rationalization. e.g. factor out `dummyMintDistributionVX`.

Reply:
done.